.changeset/clever-eagles-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-node': patch
+---
+
+precompress assets and prerendered pages (html,js,json,css,svg,xml)
```

Comment:
```suggestion
compress assets and prerendered pages (html,js,json,css,svg,xml)
```

Reply:
"Precompress" is the correct term in this location, since "compress" could mean runtime compression

Reply:
fair enough